### PR TITLE
Add support for class selector

### PR DESCRIPTION
**Pull request by adela-istrate-hs**

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
The data returned by WebDriverIO is an Array only if the selector is found more than one time. It doesn't matter if it is a class, id, li, etc.

@valentin-radulescu-hs @flavius-tirnacop-hs review?

File: lib/adapters/webdriverio.js [hunk 1]
```diff
@@ -26,10 +26,13 @@ WebDriverIOAdaptor.prototype = objectAssign({}, Browser, {
 
   getBoundingClientRect: function(selector, callback) {
     var webdriverio = this._webdriverio;
-
```
Comment (valentin-radulescu-hs) [inline lib/adapters/webdriverio.js]: Can you add this newline back? It's easier to read if the var statement and the code below are separated.

**Comment by valentin-radulescu-hs** [comment]
LGTM

File: lib/adapters/webdriverio.js (hunk 1 above)
Comment (adela-istrate-hs) [inline lib/adapters/webdriverio.js]: Done.

**Comment by flavius-tirnacop-hs** [comment]
LGTM

File: lib/adapters/webdriverio.js [hunk 2]
```diff
@@ -30,6 +30,10 @@ WebDriverIOAdaptor.prototype = objectAssign({}, Browser, {
     webdriverio.getElementSize(selector)
         .then(function(size) {
 
+          if (size instanceof Array) {
```
Comment (andrei-picus-hs) [inline lib/adapters/webdriverio.js]: Use [`Array.isArray()`](http://devdocs.io/javascript/global_objects/array/isarray) instead. It's more readable, and apparently [safer](http://web.mit.edu/jwalden/www/isArray.html).

Comment (andrei-picus-hs) [inline lib/adapters/webdriverio.js]: Also, please add a test for this.

**Pull request merged**